app/qml/PositionMarker.qml

```diff
@@ -32,12 +49,12 @@ Item {
         id: direction
         source: "gps_direction.svg"
         fillMode: Image.PreserveAspectFit
-        rotation: positionKit.direction
```

Comment:
lets stick with the terminology and create positionMarker.direction

I would prefer if ALL logic is inside the onTriggered() method  and the rest of the code would just use positionMarker.direction instead of positionKit.direction 

logic could be 
```
if (compass.reading)
  dir = compass.reading.azimuth
else 
  dir = positionKit.direction

if (abs(new - old) > 2) positionMarker.direction  = dir 
```